assets/js/components/coverage/Constants.js

```diff
@@ -107,7 +107,7 @@ export const getColumns = (
               selectHotspotAddress(record.hotspot_address);
             }}
           >
-            {hotspot_alias || startCase(data)}
+            {hotspot_alias ? `${hotspot_alias} (${startCase(data)})` : startCase(data)}
```

Comment:
if we want to sort by alias, we need a new column on the table, otherwise it gets too complicated to reason about